{"code":"42P01","file":"parse_relation.c","length":105,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"users\" does not exist","name":"error","position":"128","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:81:19","timestamp":"2025-03-28 11:01:51"}
{"level":"error","message":"Ошибка при получении пользователей: relation \"users\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:01:51"}
{"level":"error","message":"Ошибка при получении настроек: settings is not iterable","service":"salon-api","timestamp":"2025-03-28 11:01:52"}
{"code":"42P01","file":"parse_relation.c","length":105,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"users\" does not exist","name":"error","position":"128","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:81:29","timestamp":"2025-03-28 11:02:03"}
{"level":"error","message":"Ошибка при получении пользователей: relation \"users\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:02:03"}
{"code":"42P01","file":"parse_relation.c","length":105,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"users\" does not exist","name":"error","position":"128","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:81:29","timestamp":"2025-03-28 11:02:37"}
{"level":"error","message":"Ошибка при получении пользователей: relation \"users\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:02:37"}
{"code":"42P01","file":"parse_relation.c","length":105,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"users\" does not exist","name":"error","position":"128","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"users\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:81:29","timestamp":"2025-03-28 11:03:53"}
{"level":"error","message":"Ошибка при получении пользователей: relation \"users\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:03:53"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:30:03"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:30:03"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:30:26"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:30:26"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:30:47"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:30:47"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:31:12"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:31:12"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:31:24"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:31:24"}
{"code":"42703","file":"parse_relation.c","length":110,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"is_active\" does not exist","name":"error","position":"130","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"is_active\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:96:31","timestamp":"2025-03-28 11:31:46"}
{"level":"error","message":"Ошибка при получении пользователей: column \"is_active\" does not exist","service":"salon-api","timestamp":"2025-03-28 11:31:46"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:42:57"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:42:57"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:42:58"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:42:58"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:51:21"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:51:21"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:51:22"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:51:22"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:11"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:11"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:27"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:27"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:28"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Error fetching employee performance: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:462:35","timestamp":"2025-03-28 11:52:28"}
{"code":"23502","column":"name","detail":"Failing row contains (1, 1, null, Анна, Иванова, null, Старший парикмахер, {\"email\": \"\", \"phone\": \"+7 (999) 111-22-34\"}, {\"1\": {\"end\": \"18:00\", \"start\": \"09:00\"}, \"2\": {\"end\": \"18:00\", ..., t, 2025-03-27 21:44:09.861173+00, 2025-03-28 10:29:06.581414+00).","file":"execMain.c","length":453,"level":"error","line":"1982","message":"Update employee error: null value in column \"name\" of relation \"employees\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"name\" of relation \"employees\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:727:36","table":"employees","timestamp":"2025-03-28 13:29:05"}
{"code":"23502","column":"name","detail":"Failing row contains (1, 1, null, Анна, Иванова, null, Старший парикмахер, {\"email\": \"\", \"phone\": \"+7 (999) 111-22-34\"}, {\"1\": {\"end\": \"18:00\", \"start\": \"09:00\"}, \"2\": {\"end\": \"18:00\", ..., t, 2025-03-27 21:44:09.861173+00, 2025-03-28 10:55:21.7393+00).","file":"execMain.c","length":451,"level":"error","line":"1982","message":"Update employee error: null value in column \"name\" of relation \"employees\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"name\" of relation \"employees\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:727:36","table":"employees","timestamp":"2025-03-28 13:55:20"}
{"code":"22P02","file":"enum.c","length":164,"level":"error","line":"129","message":"Database error during employee update: invalid input value for enum user_role: \"Старший парикмахер\"","name":"error","routine":"enum_in","service":"salon-api","severity":"ERROR","stack":"error: invalid input value for enum user_role: \"Старший парикмахер\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:753:36","timestamp":"2025-03-28 14:04:29","where":"unnamed portal parameter $2 = '...'"}
{"code":"22P02","file":"enum.c","length":164,"level":"error","line":"129","message":"Update employee error: invalid input value for enum user_role: \"Старший парикмахер\"","name":"error","routine":"enum_in","service":"salon-api","severity":"ERROR","stack":"error: invalid input value for enum user_role: \"Старший парикмахер\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:753:36","timestamp":"2025-03-28 14:04:29","where":"unnamed portal parameter $2 = '...'"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\appointments.js:85:22","timestamp":"2025-03-28 18:06:02"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Get upcoming appointments error: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\appointments.js:85:22","timestamp":"2025-03-28 18:06:02"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Database query error (attempt 1/5): relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\appointments.js:85:22","timestamp":"2025-03-28 18:06:03"}
{"code":"42P01","file":"parse_relation.c","length":107,"level":"error","line":"1449","message":"Get upcoming appointments error: relation \"reviews\" does not exist","name":"error","position":"406","routine":"parserOpenTable","service":"salon-api","severity":"ERROR","stack":"error: relation \"reviews\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\appointments.js:85:22","timestamp":"2025-03-28 18:06:03"}
{"code":"42703","file":"parse_relation.c","length":105,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column s.phone does not exist","name":"error","position":"33","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column s.phone does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\favorites.js:10:22","timestamp":"2025-03-28 18:14:04"}
{"code":"42703","file":"parse_relation.c","length":105,"level":"error","line":"3716","message":"Get favorite salons error: column s.phone does not exist","name":"error","position":"33","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column s.phone does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\favorites.js:10:22","timestamp":"2025-03-28 18:14:04"}
{"code":"42703","file":"parse_relation.c","length":105,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column s.phone does not exist","name":"error","position":"33","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column s.phone does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\favorites.js:10:22","timestamp":"2025-03-28 18:14:05"}
{"code":"42703","file":"parse_relation.c","length":105,"level":"error","line":"3716","message":"Get favorite salons error: column s.phone does not exist","name":"error","position":"33","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column s.phone does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\favorites.js:10:22","timestamp":"2025-03-28 18:14:05"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:12","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:12","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:13","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:13","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:16","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:16","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:38","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:38","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:39","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:58:39","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 11:59:18","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 11:59:18","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 11:59:23","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 11:59:23","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:59:25","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:59:25","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:59:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 11:59:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 12:00:10","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 12:00:10","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 12:00:12","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee services: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:361:36","timestamp":"2025-04-08 12:00:12","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:14:51","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:14:51","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:14:52","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:14:52","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:16:57","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:16:57","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:16:58","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:16:58","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:17:10","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:17:10","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:17:33","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error fetching employee performance: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:435:33","timestamp":"2025-04-08 12:17:33","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-08 14:36:03","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error updating employee schedule: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-08 14:36:03","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-08 14:36:14","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error updating employee schedule: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-08 14:36:14","where":"unnamed portal parameter $1 = '...'"}
{"level":"error","message":"Неверный формат working_hours для салона #1:","service":"salon-api","timestamp":"2025-04-13 14:19:58"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-13 15:26:45","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error updating employee schedule: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-13 15:26:45","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-13 15:36:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":152,"level":"error","line":"616","message":"Error updating employee schedule: invalid input syntax for type integer: \"undefined\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"undefined\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:957:36","timestamp":"2025-04-13 15:36:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","level":"error","message":"Database query error (attempt 1/5): getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","service":"salon-api","stack":"Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:41:22)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\salons.js:13:20\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\utils\\routeHelpers.js:16:7","syscall":"getaddrinfo","timestamp":"2025-04-14 08:59:24"}
{"error":"Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:41:22)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\salons.js:13:20\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\utils\\routeHelpers.js:16:7","level":"error","message":"API Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","method":"GET","route":"/api/salons?_t=1744610364944","service":"salon-api","timestamp":"2025-04-14 08:59:24"}
{"code":"ENOTFOUND","errno":-3008,"hostname":"ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","level":"error","message":"Database query error (attempt 1/5): getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","service":"salon-api","stack":"Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:41:22)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\salons.js:13:20\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\utils\\routeHelpers.js:16:7","syscall":"getaddrinfo","timestamp":"2025-04-14 08:59:25"}
{"error":"Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg-pool\\index.js:45:11\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:41:22)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\salons.js:13:20\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\utils\\routeHelpers.js:16:7","level":"error","message":"API Error: getaddrinfo ENOTFOUND ep-old-shadow-a2poxby8-pooler.eu-central-1.aws.neon.tech","method":"GET","route":"/api/salons?_t=1744610365980","service":"salon-api","timestamp":"2025-04-14 08:59:25"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:15","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:15","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:17","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:18","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:18","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:27","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:27","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:28","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:28","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:35","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:35","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:05:36","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:06:57","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:06:57","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Database query error (attempt 1/5): invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:07:16","where":"unnamed portal parameter $1 = '...'"}
{"code":"22P02","file":"numutils.c","length":153,"level":"error","line":"616","message":"Get service error: invalid input syntax for type integer: \"categories\"","name":"error","routine":"pg_strtoint32_safe","service":"salon-api","severity":"ERROR","stack":"error: invalid input syntax for type integer: \"categories\"\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:76:22","timestamp":"2025-04-14 23:07:16","where":"unnamed portal parameter $1 = '...'"}
{"code":"42703","file":"analyze.c","length":134,"level":"error","line":"2528","message":"Ошибка при создании таблицы категорий услуг: column \"category_id\" of relation \"services\" does not exist","name":"error","position":"47","routine":"transformUpdateTargetList","service":"salon-api","severity":"ERROR","stack":"error: column \"category_id\" of relation \"services\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async addServiceCategoriesTable (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\tools\\add_service_categories_table.js:118:11)","timestamp":"2025-04-14 23:32:17"}
{"code":"23502","column":"first_name","detail":"Failing row contains (6, 1, как ка, null, null, employee, null, {\"email\": \"zhuravelof@vk.com\", \"phone\": \"89773464593\"}, {}, t, 2025-04-15 07:31:13.818987+00, 2025-04-15 07:31:13.818987+00).","file":"execMain.c","length":376,"level":"error","line":"1982","message":"Create employee error: null value in column \"first_name\" of relation \"employees\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"first_name\" of relation \"employees\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:667:36","table":"employees","timestamp":"2025-04-15 10:31:14"}
{"code":"23502","column":"first_name","detail":"Failing row contains (7, 1, как ка, null, null, employee, null, {\"email\": \"zhuravelof@vk.com\", \"phone\": \"89773464593\"}, {}, t, 2025-04-15 07:31:35.332852+00, 2025-04-15 07:31:35.332852+00).","file":"execMain.c","length":376,"level":"error","line":"1982","message":"Create employee error: null value in column \"first_name\" of relation \"employees\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"first_name\" of relation \"employees\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:667:36","table":"employees","timestamp":"2025-04-15 10:31:35"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Database error during employee update: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:874:9","table":"employee_services","timestamp":"2025-04-15 11:13:19"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Update employee error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:874:9","table":"employee_services","timestamp":"2025-04-15 11:13:19"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Database error during employee update: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:875:9","table":"employee_services","timestamp":"2025-04-15 11:18:19"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Update employee error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:875:9","table":"employee_services","timestamp":"2025-04-15 11:18:19"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Database error during employee update: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:875:9","table":"employee_services","timestamp":"2025-04-15 11:21:47"}
{"code":"23502","column":"service_id","detail":"Failing row contains (1, null).","file":"execMain.c","length":229,"level":"error","line":"1982","message":"Update employee error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint","name":"error","routine":"ExecConstraints","schema":"public","service":"salon-api","severity":"ERROR","stack":"error: null value in column \"service_id\" of relation \"employee_services\" violates not-null constraint\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:875:9","table":"employee_services","timestamp":"2025-04-15 11:21:47"}
{"code":"42703","file":"parse_target.c","length":126,"level":"error","line":"1065","message":"Create employee error: column \"name\" of relation \"employees\" does not exist","name":"error","position":"47","routine":"checkInsertTargets","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" of relation \"employees\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:677:36","timestamp":"2025-04-15 12:00:54"}
{"code":"42703","file":"parse_target.c","length":126,"level":"error","line":"1065","message":"Create employee error: column \"name\" of relation \"employees\" does not exist","name":"error","position":"47","routine":"checkInsertTargets","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" of relation \"employees\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\employees.js:677:36","timestamp":"2025-04-15 12:01:01"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:26"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:26"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:27"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:27"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:31"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:31"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:32"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:02:32"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:03:01"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:03:01"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Database query error (attempt 1/5): column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:03:07"}
{"code":"42703","file":"parse_relation.c","hint":"Perhaps you meant to reference the column \"s.name\".","length":158,"level":"error","line":"3723","message":"Get service error: column e.name does not exist","name":"error","position":"101","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column e.name does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\services.js:212:22","timestamp":"2025-04-15 12:03:07"}
{"code":"42703","file":"parse_relation.c","length":104,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"name\" does not exist","name":"error","position":"37","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:88:33","timestamp":"2025-04-15 12:22:56"}
{"level":"error","message":"Ошибка при получении пользователей: column \"name\" does not exist","service":"salon-api","timestamp":"2025-04-15 12:22:56"}
{"code":"42703","file":"parse_relation.c","length":104,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"name\" does not exist","name":"error","position":"37","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:88:33","timestamp":"2025-04-15 12:23:13"}
{"level":"error","message":"Ошибка при получении пользователей: column \"name\" does not exist","service":"salon-api","timestamp":"2025-04-15 12:23:13"}
{"code":"42703","file":"parse_relation.c","length":104,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"name\" does not exist","name":"error","position":"37","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:88:33","timestamp":"2025-04-15 12:23:16"}
{"level":"error","message":"Ошибка при получении пользователей: column \"name\" does not exist","service":"salon-api","timestamp":"2025-04-15 12:23:16"}
{"code":"42703","file":"parse_relation.c","length":104,"level":"error","line":"3716","message":"Database query error (attempt 1/5): column \"name\" does not exist","name":"error","position":"37","routine":"errorMissingColumn","service":"salon-api","severity":"ERROR","stack":"error: column \"name\" does not exist\n    at C:\\Users\\Сергей\\Documents\\slaon_app\\node_modules\\pg\\lib\\client.js:545:17\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async Object.executeQuery [as query] (C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\config\\database.js:52:18)\n    at async C:\\Users\\Сергей\\Documents\\slaon_app\\backend\\src\\routes\\settings.js:88:33","timestamp":"2025-04-15 12:32:46"}
{"level":"error","message":"Ошибка при получении пользователей: column \"name\" does not exist","service":"salon-api","timestamp":"2025-04-15 12:32:46"}
